cmd/internal/ec2/add.go

```diff
@@ -198,9 +198,12 @@ func configureSystemdFiles(projectID string) error {
 	printer.Infof(message + "\n")
 	reportStep(message)
 
-	err := checkReconfiguration()
-	if err != nil {
-		return err
+	// Prompt user if --force flag is not set
+	if !forceOverwrite {
+		err := checkReconfiguration()
```

Comment:
I think this is OK, but I wonder if there is value in logging whether we are reconfiguring or doing a fresh install.  (I.e, move the check for forceOverwrite down around just the user prompt.)